uiowa/uiowa
# Create MAUI module with configurable academic dates block.

Resolves #2714.

### Testing
- `blt dsa` a site.
- Place the `Academic dates` block on a page somewhere.
- Test out the heading size field.
- Test out the exposed filters: session and category. The date list should update after changing either of them.
  - Do this with multiple blocks placed on the same page.
- Edit the block and select a session and category for pre-filtered data. This should hide the exposed filters.
- Test out the `@session` placeholder text you can use in the headline field.
- Test out the Hide Session Badges style option.
- Review the addition of the HeadlineHelper class and the changes in other spots:
  - uids_base_preprocess_paragraph()
  - sitenow_people
  - sitenow_events
  - sitenow_articles
  - The actual field itself in layout_builder_custom/
- Review code changes.
